cppcache/integration-test/testThinClientSecurityAuthorization.cpp

```diff
@@ -54,13 +54,15 @@ const std::string locHostPort =
 std::shared_ptr<CredentialGenerator> credentialGeneratorHandler;
 
 std::string getXmlPath() {
-  char xmlPath[1000] = {'\0'};
-  const char *path = std::getenv("TESTSRC");
-  ASSERT(path != nullptr,
+  std::string path = std::string(std::getenv("TESTSRC"));
```

Comment:
Use `auto` on the lefthand side whenever you can. Almost all the new code should be corrected to use auto for all these local variables.

Reply:
N/A. Test deleted.